script.js

```diff
@@ -3,47 +3,47 @@ var buttons = document.getElementsByClassName("button");
 
 Array.prototype.forEach.call(buttons, function (button) {
   button.addEventListener("click", function () {
-    console.log(" Button text content", button.textContent);
+    console.log(" Button text content", button.textContent.trim());

```

Comment:
please extract `button.textContent.trim()` to a variable and use it on all the places.

`const buttonValue = button.textContent.trim();`

Reply:
That just a console log no need to assign a variable for that.

Reply:
And the other n times in this code block?

Reply:
Ok i got your concern.